src/include/execution/executors/insert_executor.h

```diff
@@ -34,16 +34,15 @@ class InsertExecutor : public AbstractExecutor {
    * @param child_executor the child executor to obtain insert values from, can be nullptr
    */
   InsertExecutor(ExecutorContext *exec_ctx, const InsertPlanNode *plan,
-                 std::unique_ptr<AbstractExecutor> &&child_executor)
-      : AbstractExecutor(exec_ctx) {}
+                 std::unique_ptr<AbstractExecutor> &&child_executor);
 
-  const Schema *GetOutputSchema() override { return plan_->OutputSchema(); }
```

Comment:
It is okay to leave this method in the header file because it's not something the students will have to implement.